.pre-commit-config.yaml

```
@@ -75,6 +75,35 @@ repos:
   hooks:
   - id: forbid-html-img-without-alt-text
 
+- repo: https://github.com/pre-commit/mirrors-mypy
+  rev: v0.910-1
+  hooks:
+  - id: mypy
+    additional_dependencies:
+    - types-backports
+    - types-pyOpenSSL
+    - types-requests
+    - types-six
+    args:
+    - --install-types
+    - --non-interactive
+    - --python-version=3.5
+    - cheroot/
+    pass_filenames: false
+  - id: mypy
+    additional_dependencies:
+    - types-backports
```

The error message at https://results.pre-commit.ci/run/github/16620627/1638950716.FddrDhgSRnSAcxDS8WIZng says that this is necessary under Python 2:
```suggestion
    - typed-ast
    - types-backports
```